trino/trino_test.go

```diff
@@ -249,6 +249,82 @@ func TestQueryForUsername(t *testing.T) {
 	}
 }
 
+type TestQueryProgressCallback struct{}
+
+func (*TestQueryProgressCallback) ProgressUpdated(qr QueryProgressCallbackResponse) {
+	fmt.Printf("%+v, %+v \n", qr.QueryStats.State, qr.QueryStats.ProgressPercentage)
```

Comment:
You shouldn't print anything here. This implies you rely on a human to do the assert. Instead, you should increment some counter, and do a regular assert of its value at the end of the test.